insights/parsers/lsvmbus.py

```diff
@@ -84,3 +88,32 @@ def parse_content(self, content):
                     'target_cpu': patrn_targetcpu.search(part[3]).groups()[0],
                 }
             )
+
+
+@parser(Specs.lsvmbus)
+class LsVMBus(Parser, list):
```

Comment:
@psachin why create a new parser?  Why wouldn't you just collect the data in the same format and use the existing parser?  Wouldn't the new parser be limited in that its only collecting a subset of the data?

Reply:
@bfahr 

>  why create a new parser? Why wouldn't you just collect the data in the same format and use the existing parser? 

Making the data compatible with the existing parser would be overwork(I think)

Do you prefer to make the data compatible to the existing parser?

>    Wouldn't the new parser be limited in that its only collecting a subset of the data?

Only the essential data is collected now. The important values are the IDs & device type. However, it can be extended if the future rules demands that.